woocommerce/woocommerce
# Add coming soon social links template part

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes https://github.com/woocommerce/woocommerce/issues/52397.


This PR extracts social UI blocks into a reusable template part for coming soon templates and replaces existing social blocks in the coming soon templates with the new reusable template part.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a WooCommerce site
2. Go through the Core Profiler onboarding wizard
3. Click on the "Coming Soon" badge on the admin bar
4. Click on the link that takes you to the site editor
5. Verify that the social links is displayed
6. Go to `Tools > WCA Test Helper > Features` and activate `coming-soon-newsletter-template` feature flag
7. Go to coming soon template site editor page
8. In site editor, expand the `Design` tab and select `Default Coming Soon` template
9. Verify that the social links is displayed

![Screenshot 2024-11-15 at 16 41 00](https://github.com/user-attachments/assets/7b2e097d-201e-408b-aec4-0983cab4c2f6)
![Screenshot 2024-11-15 at 16 41 46](https://github.com/user-attachments/assets/f2e078b2-6e6f-40d2-87a6-09c3127d09b5)




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
